packages/common/src/ui/layout/tables/columns/types.ts

```diff
@@ -76,5 +76,5 @@ export interface Column<T extends DomainObject> {
 
 export interface ColumnDefinition<T extends DomainObject>
   extends Partial<Omit<Column<T>, 'key'>> {
-  key: keyof T | GenericColumnKey;
+  key: keyof T | GenericColumnKey | string;
```

Comment:
and again. Realising that restricting a column to a key that must be on the object is not ideal. Related entities, the `Select` column and any other 'random' cells we need to add will break this rule and there's no real reason for it, other than the `DataAccessor` returning a 'valid' value as a default. Maybe this could be extended to be something like.. either the `key` is a `keyof T` OR, `DataAccessor` must be supplied?

Reply:
the `DataAccessor` would be a good option, though I can't quite see how that is possible. `string` good for now..